.github/workflows/delivery/chocolatey/pack.nuspec

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>pack</id>
+    <version>{{PACK_VERSION}}</version>
+    <packageSourceUrl>https://github.com/buildpacks/pack</packageSourceUrl>
+    <owners>Cloud Native Buildpack Authors</owners>
+
+    <title>pack</title>
+    <authors>Cloud Native Buildpack Authors</authors>
+    <projectUrl>https://github.com/buildpacks/pack</projectUrl>
+    <iconUrl>https://rawcdn.githack.com/buildpacks/artwork/36f02fae98ed82bb3175918796b6cca7acb813df/light-background/logo-light.png</iconUrl>
+    <licenseUrl>https://github.com/buildpacks/pack/blob/main/LICENSE</licenseUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <projectSourceUrl>https://github.com/buildpacks/pack</projectSourceUrl>
+    <docsUrl>https://buildpacks.io/docs/</docsUrl>
+    <mailingListUrl>cncf-buildpacks@lists.cncf.io</mailingListUrl>
```

Comment:
Sorry for not being specific but I think this should be the url: https://lists.cncf.io/g/cncf-buildpacks

It aligned with the initial intent proposed here: https://github.com/chocolatey/choco/issues/205#issuecomment-115306421

Reply:
Ah yes, of course. I misread it as the mailing list email.